blog/2021-09-07-introducing-capact.md

```diff
@@ -0,0 +1,124 @@
+---
+slug: introducing-capact
+title: Introducing Capact, your new way to manage applications and infrastructure
+authors:
+  - pkosiec
+tags:
+  - capact
+  - introduction
+  - kubernetes
+  - release
+  - get-started
+  - release-sneak-peek
+---
+
+Managing the lifecycle of infrastructure, applications and processes in modern IT is problematic. 
+There are a variety of tools and best practices, which are ever-changing. It's a struggle to stay up to date with these practices, and not everyone is an expert in everything. Technical debt is inevitable.
+
+We are ultimately all alone. Siloed within the context of team, department or company, locked-in in a particular ecosystem of tooling.  To deliver end-to-end capabilities, we build them from a scratch in a vacuum. We consume API calls, transform data, build and manage infrastructure and applications. Do we all really have different use cases? Don't we spend too much time on solving the same problems as others have?
+
+What if we could share the expertise, and, in a result, save our time? While collaborating together, we ensure we use the best tools out there, following best practices. This is something that already happens - we have plenty of libraries and frameworks out there. But what if we went a step further and have a way to create, use and share building blocks that are language-agnostic abstracted capabilities?
+
+For example, if you're not a cloud expert, maybe all you need to know is that you want a managed PostgreSQL database on AWS, and let others make it happen to provision AWS RDS for PostgreSQL using Terraform. If you need any Kubernetes 
+cluster - maybe by saying "I want any Kubernetes cluster" and letting magic happen, you could cover both local development (e.g. using [K3d](https://k3d.io)) and production (e.g. Google Cloud GKE) scenarios...
```

Comment:
Overall, what you want to express by those sentence is good, but for me those sentences are too long to read it at once.